alpha/apps/kaltura/lib/batch2/kFlowHelper.php

```diff
@@ -251,6 +256,24 @@ public static function handleImportFinished(BatchJob $dbBatchJob, kImportJobData
 		return $dbBatchJob;
 	}
 
+	private function excuteClip($flavorAsset, $dbBatchJob) {
```

Comment:
rename function to public static kBusinessPreConvertDL::convertSource($flavorAsset, $dbParentBatchJob=null)